### ISSUE TITLE
Unable to reproduce results.

### ISSUE THREAD

**Issue by MediciPrime** [rr1859/R.4Cker#15]
@rr1859 I am trying to reproduce the bedgraphs from the following data:
1.  Igk_MiEk_DN_WT_1 & Igk_MiEk_DN_WT_2, Igk_MiEk_ImmB_WT_1 & Igk_MiEk_ImmB_WT_2
2.  Tcrb_Eb_DN_WT_1 & Tcrb_Eb_DN_WT_2, Tcrb_Eb_ImmB_WT_1 & Tcrb_Eb_ImmB_WT_2

The names for the raw data were taken from the 4C-ker supplemental table.  I was able to find the raw data for '2', but the raw data for '1' doesn't seem to exist.  Now for '2' I ran fastqc to double check the primer and restriction enzyme sites but the 'per base sequence content' tables' results were inconclusive.  Please check [EbDN_1](https://github.com/MediciPrime/Repository-Images/blob/master/ebdn_1.png), [EbDN_2](https://github.com/MediciPrime/Repository-Images/blob/master/ebdn_2.png), [EbImmB_1](https://github.com/MediciPrime/Repository-Images/blob/master/ebimmb_1.png), and [EbImmB_2](https://github.com/MediciPrime/Repository-Images/blob/master/ebimmb_2.png).  Even though the supplemental table states that the primer is 'TGTGGATTGATTAAGCCATG' these fastqc results don't seem to agree.  

Assuming that the primer sequence is correct and the final four bases of the primer represent the restriction enzyme NlaIII, would you agree with a 'fragment length' of 20?  
Also where would I be able to find the raw data for '1'?

**Comment by rr1859** [comment]
The read length for those samples are 101bp however for this analysis I only used 51bp. The first 6bp are the barcode, followed by the primer and then the interacting fragment. So you need to trim 26bps from the 5' end and the map the rest of the read or only the next 25 bps. Im not sure why the Igk_MiEK baits are not in GEO but you can find them here - https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80272

**Comment by MediciPrime** [comment]
@rr1859 thank you for your reply, I was able to get the data I needed.  Now my main objective is to try and figure out the adaptive windows sizes at 10 kb from the bait using NlaIII as the first restriction enzyme.  I understand that the answer can depend on the activity of the bait, quality of the sequencing and percent of aligned reads.  But assuming nominal or even optimal conditions would the adaptive window sizes ever drop at or below 700bp when they are 10kb away from the bait? 

I tried to figure out the answer to this question on my own but I have been running into a few problems.  I will highlight the problems in my next post.
